app/src/main/java/com/example/cleanarchitecturesample/presentation/ui/ArticlesAdapter.kt

```diff
@@ -47,8 +46,8 @@ class ArticlesAdapter : RecyclerView.Adapter<ArticlesAdapter.ViewHolder>() {
                 tvTitle.text = article.title
                 tvDescription.text = article.summary
                 tvPublishedAt.text = article.publishedAt
-                    ?.toDate(ARTICLE_DATE_INPUT_FORMAT)
-                    ?.formatTo(DATE_OUTPUT_FORMAT) ?: ""
+                    ?.toDate(Constants.ARTICLE_DATE_INPUT_FORMAT)
+                    ?.formatTo(Constants.DATE_OUTPUT_FORMAT) ?: ""
```

Comment:
Nit - We can also use `orEmpty()` instead of `?: ""` to achieve the same output
Example: `?.formatTo(Constants.DATE_OUTPUT_FORMAT).orEmpty()`

Reply:
sure we can, will do it.